low-level/sensors/impl/generic/node_hw.py

```diff
@@ -33,7 +33,8 @@
 from framework.base.internal_msgQ import InternalMsgQ
 from framework.base.module_thread import SensorThread
 from framework.base.sspl_constants import (
-    DATA_PATH, BMCInterface, PRODUCT_FAMILY, ServiceTypes, node_key_id)
+    DATA_PATH, BMCInterface, PRODUCT_FAMILY, ServiceTypes, node_key_id,
+    DEFAULT_RECOMMENDATION)
```

Comment:
`DEFAUTL_RECOMMENDATION` is also not needed now.

Reply:
removed unused var